### ISSUE TITLE
Feature Request: Change order of reference level in ggcoef_compare()

### ISSUE THREAD

**Issue by jerrodanzalone** [larmarange/ggstats#23]
I find your ggocef_compare() such a useful tool for comparing models. Is there a way to retain the factor ordering for the reference levels? 

    d <- as.data.frame(Titanic)

    d$Sex <- factor(d$Sex, levels=c("Male", "Female"))
    d$Sex=relevel(as.factor(d$Sex),ref="Male")
    
    m1 <- glm(Survived ~ Sex + Age, family = binomial, data = d)
    m2 <- glm(Survived ~ Sex + Age + Class, family = binomial, data = d)
    
    models <- list("Model 1" = m1, "Model 2" = m2)
    
    ggcoef_compare(models, exponentiate = TRUE)

Which produces: 

![gg_compare](https://user-images.githubusercontent.com/41122083/226828889-ab67a4a6-6e9e-414a-8b93-3bf649cd1af9.png)

This differs from ggcoef_model(), which retains the specified order: 

    d <- as.data.frame(Titanic)

    d$Sex <- factor(d$Sex, levels=c("Male", "Female"))
    d$Sex=relevel(as.factor(d$Sex),ref="Male")
    
    m1 <- glm(Survived ~ Sex + Age, family = binomial, data = d)    
    
    ggcoef_model(m1, exponentiate = TRUE)

![ggcoef_model](https://user-images.githubusercontent.com/41122083/226829176-c36a5902-5c91-4bb8-b8d0-f2c41a324842.png)

There is probably a straightforward way to do this, but I'm struggling to find it. Additionally, if I try to remove the reference row, I receive an error: 

    d <- as.data.frame(Titanic)

    d$Sex <- factor(d$Sex, levels=c("Male", "Female"))
    d$Sex=relevel(as.factor(d$Sex),ref="Male")
    
    m1 <- glm(Survived ~ Sex + Age, family = binomial, data = d)
    m2 <- glm(Survived ~ Sex + Age + Class, family = binomial, data = d)
    
    models <- list("Model 1" = m1, "Model 2" = m2)
    
    ggcoef_compare(models, exponentiate = TRUE, add_reference_rows = FALSE)
    ```
>  Error in eval_tidy(dot, data = mask): object 'reference_row' not found

Thanks!

**Comment by larmarange** [comment]
It should be fixed by #24

**Comment by jerrodanzalone** [comment]
@larmarange Fantastic. Thank you!
